travvy/src/components/Onboarding.vue

```diff
@@ -119,7 +119,21 @@ export default {
       console.log(this.instagram_username)
       console.log(this.preferred_activity_type)
       console.log(this.errors)
-
+      // sets the activity id based on the activity type
+      // recommends attractions based on activity id
+         if(this.preferred_activity_type == "Active"){
+            this.activity_id = 111
+        } else if (this.preferred_activity_type == "Food"){
+            this.activity_id = 112
+        }else if (this.preferred_activity_type == "Entertainment"){
+           this.activity_id = 113
+        } else if (this.preferred_activity_type == "Historical"){
+            this.activity_id = 114
+        } else if (this.preferred_activity_type == "Family"){
+            this.activity_id = 115
+        } else if (this.preferred_activity_type == "Nature"){
+            this.activity_id = 116
+        }
```

Comment:
Is this a workaround to querying the database for these values?

Reply:
These values are currently hard-coded yes. Ideally, they will pull from the database along with the preferred activity type in onboarding. This improvement could be made in the future, but still provides the same functionality now!